src/utils/components/DiskModal/state/initialState.ts

```diff
@@ -16,6 +16,7 @@ export type DiskFormState = {
   volumeMode: string;
   storageClassProvisioner: string;
   enablePreallocation: boolean;
+  useAsBootSource: boolean;
```

Comment:
name starting as `use...` might be a bit confusing since it's not a hook, maybe change it to `asBootSource` or something similar?